backend/server.ts

```diff
@@ -23,4 +24,10 @@ app.get("/api/users", async (req: Request, res: Response) => {
   res.send(users)
 })
 
+//get all Gardens
+app.get("/api/gardens", async (req:Request , res:Response) => {
```

Comment:
I asked Drew to update this too, but I decided to change the endpoint from `/api/gardens` to just `/gardens` to follow best practice. Thanks for the work Storm!